resources/views/filament/pages/dashboard.blade.php

```diff
@@ -0,0 +1,100 @@
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Aiding App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    <https://www.canyongbs.com> or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+--}}
+@php
+    use AidingApp\Authorization\Filament\Widgets\UnlicensedNotice;
+    use App\Filament\Widgets\ListServiceRequestTableWidgets;
+    use App\Filament\Widgets\ServiceRequestDonutChart;
+    use App\Filament\Widgets\ServiceRequestLineChart;
+    use App\Filament\Widgets\ServiceRequestWidget;
+@endphp
+
+<x-filament-panels::page>
+    <div class="grid gap-6">
+        <div
+            class="col-span-full flex flex-col items-center rounded-lg bg-black bg-cover bg-no-repeat px-16 py-8 lg:col-span-5"
+            style="background-image: url('{{ asset('images/banner.png') }}')"
+        >
+            <div class="grid w-full gap-1 text-center md:text-start md:text-3xl">
+                <div class="text-3xl font-bold text-white">
+                    Welcome,
+                </div>
+
+                <div class="text-4xl font-bold text-white">
+                    {{ auth()->user()->name }}!
+                </div>
+
+                <div class="text-xl text-gray-200">
+                    <p id="current-date"></p>
+                </div>
+
+                <div class="text-xl text-gray-200">
+                    <p id="current-time"></p>
+                </div>
+            </div>
+        </div>
+
+        <div class="col-span-full flex flex-col gap-3 lg:col-span-5">
+            @if (UnlicensedNotice::canView())
+                @livewire(UnlicensedNotice::class)
+            @else
+                @livewire(ServiceRequestWidget::class)
+
+                <div class="flex gap-3">
+                    <div class="w-full md:w-1/2">
+                        @livewire(ServiceRequestLineChart::class)
+                    </div>
+                    <div class="w-full md:w-1/2">
+                        @livewire(ServiceRequestDonutChart::class)
+                    </div>
+                </div>
+
+                @livewire(ListServiceRequestTableWidgets::class)
+            @endif
+        </div>
+    </div>
+</x-filament-panels::page>
+
+<script>
```

Comment:
I am realizing now that this date time will most likely need to be passed through PHP so that we can apply the timezone using the `DisplaySettings` class so that it is in the User or colleges timezone.

We will need to make this change in Advising App as well if we didn't do that, I'll add a ticket for it.